crates/astria-api/src/sequencer/v1alpha1/celestia.rs

```diff
@@ -0,0 +1,643 @@
+use sha2::{
+    Digest as _,
+    Sha256,
+};
+
+use super::{
+    raw,
+    IncorrectRollupIdLength,
+    RollupId,
+};
+use crate::Protobuf;
+
+/// A bundle of blobs constructed from a [`super::SequencerBlock`].
+///
+/// Consists of a head [`CelestiaSequencerBlob`] and a tail of [`CelestiaRollupBlob`]s.
+/// Used as a pass-through data structure to
+pub(super) struct CelestiaBlobBundle {
```

Comment:
Introduced this module-private type to make the conversion from block to blobs more obvious